source/docs/appendices/migration-guides.md

```diff
@@ -9,7 +9,47 @@ Our goal is to make migrations as smooth as possible. This is why we try to make
 
 ## `2.14.0` -> `2.15.0`
 
-Nothing yet!
+### Node Version Support Change
+
+We now support node versions 14, 16, 17. If you are using node 12 you will need to update your node version (node 16 is recommended).
```

Comment:
```suggestion
We now support node versions 14, 16, 17. If you are using node 12 ([which reached it's end of life](https://nodejs.org/en/about/releases/)), you must update your node version (we recommend node 16).
```

Reply:
our CI allows failures for node 17, so I don't think we can write that we support node 17

Reply:
it allows but it is not failing